RxSwift/Observables/WithLatestFrom.swift

```diff
@@ -25,6 +26,7 @@ extension ObservableType {
      Merges two observable sequences into one observable sequence by using latest element from the second sequence every time when `self` emits an element.
 
      - seealso: [combineLatest operator on reactivex.io](http://reactivex.io/documentation/operators/combinelatest.html)
+     - note: Elements emitted by self before the second source has emitted any values will be omitted.
```

Comment:
Adds this:

![image](https://user-images.githubusercontent.com/1330668/102343252-97d89880-3f9a-11eb-81a2-89c66ef2759e.png)